src/BeamAdapter/component/mapping/AdaptiveBeamMapping.inl

```diff
@@ -45,6 +45,20 @@
 #include <sofa/core/ConstraintParams.h>
 #include <sofa/core/MechanicalParams.h>
 
+// execution policies only supported by MSVC >=2019 and GCC >=10
+#ifdef _MSC_VER
+    #define HAS_SUPPORT_STL_PARALLELISM (_MSC_VER > 1921)
+#elif defined(__GNUC__)
+    #define HAS_SUPPORT_STL_PARALLELISM  (__GNUC__ > 9)
+#else
+    #define HAS_SUPPORT_STL_PARALLELISM  false
```

Comment:
I was not aware that clang did not support parallel algorithms :(

Reply:
actually, it is the libc++ which does not implement it, so it could be possible to do clang + libstdc++ but lazy to do all the checks... :/

Reply:
It seems that https://en.cppreference.com/w/cpp/compiler_support/17#C.2B.2B17_library_features
is really outdated actually (last edit june 2020 :F)
It does seem that clang11 does support execution policies